sys/dev/etherswitch/e6000sw/e6000sw.c

```diff
@@ -259,8 +260,16 @@ e6000sw_probe(device_t dev)
 		 */
 		resource_int_value(device_get_name(sc->dev),
 		    device_get_unit(sc->dev), "is8190", &is_6190);
+	(void) resource_int_value(device_get_name(sc->dev),
```

Comment:
Return value casting to void isn't necessary, unless it's explicitly has a `[[nodiscard]]` attribute.

```
-	(void) resource_int_value(device_get_name(sc->dev),
-		device_get_unit(sc->dev), "is6190x", &is_6190x);
-	if (is_6190 !=0 && is_6190x != 0)
+
+	resource_int_value(device_get_name(sc->dev),
+	    device_get_unit(sc->dev), "is6190x", &is_6190x);
+	if (is_6190 != 0 && is_6190x != 0)
```

Since both can't be available (according to it), there's no need to check for `is_6190x` if `is_6190` is true.
```
 	if (is_6190 != 0)
 		sc->swid = MV88E6190;
-	if (is_6190x != 0)
+	else if (is_6190x != 0)
 		sc->swid = MV88E6190X;
```

Reply:
Thanks a lot for your review. I've made changes based on your suggestions.